.github/workflows/release.yaml

```diff
@@ -6,6 +6,12 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - 'tests/**'
+      - renovate.json
+      - poetry.lock
+      - pyproject.toml
```

Comment:
If only package metadata in `pyproject.toml` or `poetry.lock` changes without changing `requirements.txt` we shouldn't publish.

Reply:
@carlcsaposs-canonical we will need to find a way to make release.yaml workflow shared via data-platform-workflows to share such changes across our repos (avoiding copy&paste). Sure, no rush here!

Reply:
@dragomirp should "docs" also be in this path? Or we block them on CI/CD level (skipping building).
Q: shy do we spend runners time on building if we not release? Should we [skip](https://github.com/canonical/postgresql-k8s-operator/blob/main/.github/workflows/ci.yaml#L11) building too?

Reply:
> @dragomirp should "docs" also be in this path? Or we block them on CI/CD level (skipping building). 
Docs will still execute the workflow. If we don't intend to do anything automated with them, we can add them to the ignores.

> Q: shy do we spend runners time on building if we not release? Should we [skip](https://github.com/canonical/postgresql-k8s-operator/blob/main/.github/workflows/ci.yaml#L11) building too?
This will prevent the entire pipeline from running (both CI workflow and Release jobs). In the future, when we aren't starved for runners, we may consider running just the CI flow as a sanity check, but ATM I don't think it's worth it.